### ISSUE TITLE
Convert GitHub issue references to IRI values

### ISSUE THREAD

**Issue by balhoff** [geneontology/go-ontology#22876]
We are converging on OBO-wide conventions for referencing URLs as annotation values. We would prefer to use real IRIs instead of strings with datatype `xsd:anyURI`. Need to:

- [ ] Bulk update existing values
- [ ] Update GO editor guide
- [ ] Notify GO editors

**Comment by matentzn** [comment]
We have been doing this for other ontologies with orcids though.. can you generate a minimally failing OBO example?

**Comment by balhoff** [comment]
You're right, I see that in Uberon. I'll try to figure out what's different.

**Comment by matentzn** [comment]
We did have this problems as well along the way, things to look out for:

- rdfs:seeAlso is `metadata: TRUE`
- None of the github issues is a subject in a triple
- as you are essentially creating a relation now, the GO terms are no longer connected by property-value, but by `relationship` to the github issue.

**Comment by balhoff** [comment]
The error seems to arise from IRI values containing `#` fragments, e.g., `relationship: term_tracker_item https://github.com/geneontology/go-ontology/issues/21024#issuecomment-909025243`.

**Comment by balhoff** [comment]
This is an unfortunate problem, because having a hash in an IRI is pretty common, especially for semantic web identifiers.

**Comment by matentzn** [comment]
Very unfortunate, but maybe we can tell the OBO parser to permit `#` without creating another version?

**Comment by pgaudet** [comment]
Are we still planning to to this @balhoff ?
